apm.dev.ipynb

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
Should not it be moved to pynxtools/dataconverter/readers/apm/dev or debug?
Also the whole debug directory could go there, couldn't it?

Reply:
I will even say that instead of having "debug" code. Whatever this is debugging should become test code for that same functionality. This way it is easy to confirm that it is "de-bugged" in the CI without having to run anything locally for reviewers.

Reply:
I agree that is all what this directory on my computer is about, I have collected a corpus of about 160 real world examples from atom probe groups worldwide (published research) against which I am running the converter via this directory. Some of these datasets are openly accessible and could thus indeed be placed or used for writing tests, the apm dataconverter code has been made a pynxtools plugin and that directory has been removed